### PR TITLE
New version: MusicBrainz.Picard version 2.90000.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115192&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/m/MusicBrainz/Picard/2.90000.0/MusicBrainz.Picard.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Architecture: x64
+  InstallerUrl: https://github.com/metabrainz/picard/releases/download/release-2.9/MusicBrainz-Picard-2.9.msix
+  InstallerSha256: 8F194CBB35B8FE6BEDBA9E7B204B9C6CB95C0464699023F8A531FA902A16D0DA
+  PackageFamilyName: MetaBrainzFoundationInc.org.musicbrainz.Picard_s89a1zkh1p1gp
+- Architecture: x64
+  InstallerUrl: https://github.com/metabrainz/picard/releases/download/release-2.9/MusicBrainz-Picard-2.9.msix
+  InstallerSha256: 8F194CBB35B8FE6BEDBA9E7B204B9C6CB95C0464699023F8A531FA902A16D0DA
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 2.8.5
+    ProductCode: MusicBrainz Picard
```
Comment (Trenly) [inline manifests/m/MusicBrainz/Picard/2.90000.0/MusicBrainz.Picard.installer.yaml:22]:
```suggestion
```

**Comment by Scavy** [comment]
I have made a PR with the fix to @vedantmgoyal2009 PR on his fork.
[https://github.com/vedantmgoyal2009/winget-pkgs/pull/490](https://github.com/vedantmgoyal2009/winget-pkgs/pull/490)
Since I can't do it here, and are not supposed to post a new PR here as long as this is active.
In case of no action from any of the participants, this PR is essentially blocking the release of a correct PR.

**Comment by vedantmgoyal2009** [comment]
@Scavy your PR has been merged 👍

**Comment by Scavy** [comment]
@vedantmgoyal2009 I've made yet a PR with the missing fields.
Didn't realize in the first PR that there would be fields missing as well. :)

**Review by Scavy — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Pull request merged**